### ISSUE TITLE
[Versions] Centralized date + number formatting service

### ISSUE THREAD

**Issue by markus-moser** [pimcore/studio-ui-bundle#356]
It is important to have a centralized location across pimcore studio for formatting dates and numbers.

* For dates we seem to already have something like this but the number formatting for the file size is on a special location. 

* Please let's discuss timezones together. For dates we will need some more logic to be able to handle timezones correctly (https://github.com/pimcore/pimcore/issues/16184). 

* For file sizes when we decide to calculate the formatting in the frontend we would need to implement a similar logic to `formatBytes` from the core to show it in KB, MB, GB etc. depending on the size https://github.com/pimcore/pimcore/blob/50340aa5ca234280894ce5b24db5851fd8d30856/lib/helper-functions.php#L233

Related to #349

**Comment by markus-moser** [comment]
blocked by #365

**Comment by markus-moser** [comment]
I created a separate issue to further improve the file size formatting logic: #439 -> I'll close this one.
